### PR TITLE
Added confirmation dialog

### PR DESCRIPTION
**Pull request by olegded**

### CODE REVIEW

**Comment by olegded** [comment]
@KeanW please review :-)

**Comment by olegded** [comment]
We should merge this one first. Merging was a bit of pain :-)

**Comment by olegded** [comment]
screen shot 2015-06-04 at 1 22 30 pm 2

**Comment by olegded** [comment]
screen shot 2015-06-04 at 1 22 35 pm 2

File: html/index.html [hunk 1]
```diff
@@ -4,9 +4,13 @@
         <meta charset="utf-8">
         <title>VR Party</title>
         <link rel="stylesheet" href="css/style.css" type="text/css">
+        <link rel="stylesheet" href="css/jquery-ui.min.css" type="text/css">
+        <link rel="stylesheet" href="css/jquery-ui.theme.min.css" type="text/css">
```
Comment (larsxschneider) [inline html/index.html]: Can/Should we try to load jquery from a CDN?
https://code.jquery.com/

Comment (larsxschneider) [inline html/index.html]: https://code.jquery.com/ui/1.11.4/themes/flick/jquery-ui.css
https://code.jquery.com/ui/1.11.4/jquery-ui.min.js

should do the trick

Comment (larsxschneider) [inline html/index.html]: plus:
https://code.jquery.com/jquery-2.1.4.min.js

**Pull request merged**